scripts/l2-bridge/l2-bridge-deploy-l1-multicall.sh

```diff
@@ -33,6 +33,10 @@ forge create --rpc-url "${L1_RPC_URL}" --private-key "${L1_FUNDED_PRIVATE_KEY}"
 DEPLOYED_ADDRESS=$(grep "Deployed to:" forge_output.log | awk '{print $3}')
 rm forge_output.log
 
+# Update the .env.bridge file with the deployed address
+sed -i.bak "s/^NEXT_PUBLIC_L1_MULTICALL3_ADDRESS=.*/NEXT_PUBLIC_L1_MULTICALL3_ADDRESS=${DEPLOYED_ADDRESS}/" ../.env.bridge
+rm ../.env.bridge.bak
+
 # Update the .env file with the deployed address
 sed -i.bak "s/^NEXT_PUBLIC_L1_MULTICALL3_ADDRESS=.*/NEXT_PUBLIC_L1_MULTICALL3_ADDRESS=${DEPLOYED_ADDRESS}/" ../.env
```

Comment:
should we also remove this line?

Reply:
Theoretically, we only need `NEXT_PUBLIC_L1_MULTICALL3_ADDRESS` to be saved in one file, either .env.bridge or .env. However, if both files have `NEXT_PUBLIC_L1_MULTICALL3_ADDRESS` defined, I thought we should keep them consistent. So here I'm only 'replacing' existing definition instead of 'adding' one.

Reply:
yes it should only be in .env.bridge

i meant should be remove line 40-41 in this case?

Reply:
yep, already did